docs/layers/hexagon-cell-layer.md

```diff
@@ -120,9 +120,4 @@ Method called to retrieve the color of each object. Color should be set to
 
 - Default: `object => object.elevation`
 
-Method called to retrieve the elevation of each object.
-1 unit approximate to 100 meters.
-
-
-## Source Code
-[HexagonCellLayer](https://github.com/uber/deck.gl/tree/master/src/layers/core/hexagon-cell-layer)
+Method called to retrieve the elevation of each object. 1 unit approximate to 100 meters.
```

Comment:
@heshan0131 can you check if the prop defaults (100 meter, lng.lat etc are still correct in this doc?)